app/src/components/map/DatePicker.jsx

```diff
@@ -1,74 +1,48 @@
 import React, { Component } from 'react';
-import map from '../../../styles/index.scss';
+import moment from 'moment';
+import ReactDatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
 
 class DatePicker extends Component {
   constructor(props) {
     super(props);
-    this.onDatePickerChange = this.onDatePickerChange.bind(this);
+    this.onChange = this.onChange.bind(this);
   }
 
-  onDatePickerChange(e) {
-    const start = (e.target.id === 'inner_extent_start') ? new Date(e.target.value) : this.props.start;
-    const end = (e.target.id === 'inner_extent_end') ? new Date(e.target.value) : this.props.end;
-    this.props.onDatePickerChange([start, end]);
+
+  onChange(m) {
+    // convert moment to date
+    this.props.onChange(m.clone().toDate());
   }
 
   render() {
     return (
-      <div className={map.date_inputs}>
-        <label htmlFor="inner_extent_start">
-        Start date
-          <input
-            type="date"
-            id="inner_extent_start"
-            value={this.props.start.toISOString().slice(0, 10)}
-            min={this.props.startMin.toISOString().slice(0, 10)}
-            max={this.props.startMax.toISOString().slice(0, 10)}
-            onChange={this.onDatePickerChange}
-          />
-        </label>
-        <label htmlFor="inner_extent_end">
-          End date
-          <input
-            type="date"
-            id="inner_extent_end"
-            value={this.props.end.toISOString().slice(0, 10)}
-            min={this.props.endMin.toISOString().slice(0, 10)}
-            max={this.props.endMax.toISOString().slice(0, 10)}
-            onChange={this.onDatePickerChange}
-          />
-        </label>
+      <div>
+        <ReactDatePicker
+          selected={moment(this.props.selected)}
+          minDate={moment(this.props.minDate)}
+          maxDate={moment(this.props.maxDate)}
+          onChange={this.onChange}
+        />
       </div>
     );
   }
 }
 
 DatePicker.propTypes = {
-  onDatePickerChange: React.PropTypes.func,
-  /**
-   * Current start date set (a Date object)
-   */
-  start: React.PropTypes.object,
+  onChange: React.PropTypes.func,
   /**
-   * Current end date set (a Date object)
+   * Current  date set (a Date object)
    */
-  end: React.PropTypes.object,
+  selected: React.PropTypes.object,
   /**
    * Minimum allowed start date (a Date object).
    */
-  startMin: React.PropTypes.object,
-  /**
-   * Maximum allowed start date (a Date object). Depends on the inner extent start date.
-   */
-  startMax: React.PropTypes.object,
-  /**
-   * Minimum allowed end date (a Date object). Depends on the inner extent end date.
-   */
-  endMin: React.PropTypes.object,
+  minDate: React.PropTypes.object,
```

Comment:
if this is the absolute max and min dates a user can choose, they are already declared in constants.js(x)